monitoring/healthz/src/DiscoveryTrending.tsx

```diff
@@ -1,44 +1,75 @@
 import useSWR from 'swr'
 import { SP, useDiscoveryProviders } from './useServiceProviders'
 
+const fetcher = (url: string) => fetch(url).then((res) => res.json())
+
 export function DiscoveryTrending() {
   const { data: sps, error } = useDiscoveryProviders()
-  if (error) return <div>error</div>
   if (!sps) return null
+
+  const responses = sps.map((sp) => {
+    const endpoint = sp.endpoint
+    const res = useSWR(sp.endpoint + '/v1/full/tracks/trending', fetcher)
```

Comment:
Any reason we're using this over the root `health_check`?

Reply:
that's so I can get the block directly from the endpoint, otherwise i'd have to make two calls and there's a chance they're out of sync